src/assets/svgs/calendar.ts

```diff
@@ -0,0 +1,14 @@
+export default `
```

Comment:
With `svg` that use one fill color, I generally find it best to:
- Flatten the vector in Figma so that there is no stroke information. We want the vector to have a fill only.
- Remove any `fill` declarations from the svg so we can set the fill color in the `SvgXml` props